drivers/timer/nrf_rtc_timer.c

```diff
@@ -11,6 +11,7 @@
 #include <drivers/timer/system_timer.h>
 #include <sys_clock.h>
 #include <hal/nrf_rtc.h>
+#include <drivers/nrfx_common.h>
```

Comment:
This is not needed (this file gets included through `<hal/nrf_rtc.h>`) and actually it is a bit confusing (looks like something from Zephyr's `include/drivers/` is included).

Reply:
done